R/cleanup.R

```diff
@@ -0,0 +1,7 @@
+#' Remove all packages from lockbox `library`.
```

Comment:
Can you remove the backticks? In R docs, backticks are `\code{foo}` instead by the way.